glossary/staff-definition.md

```diff
@@ -5,3 +5,9 @@ type: Resource
 # Staff
 
 Someone employed directly by the Foundation For Public Code. This excludes community members, volunteers and contractors.
+
+# Contractors
+
+Some roles in our organization are fulfilled by contractors. For transparency, these are mentioned with the legal 
+entity they bill through.
```

Comment:
I think this needs it's own file as it is a term we use separately from staff, so a `contractor-definition.md` in this folder with this definition in it.